src/modality-specific-files/eye-tracking.md

```diff
@@ -0,0 +1,233 @@
+      "CogAtlasID": "RECOMMENDED",
+      "CogPOID": "RECOMMENDED",
+      "DeviceSerialNumber": "RECOMMENDED",
+   }
+) }}
+
+Specific eye-tracking fields MUST be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "SamplingFrequency__eyetrack": "REQUIRED",
+      "SampleCoordinateUnit": "REQUIRED",
+      "SampleCoordinateSystem": "REQUIRED",
+      "EnvironmentCoordinates": "REQUIRED",
+      "ScreenSize": "REQUIRED",
+      "ScreenResolution": "REQUIRED",
+      "ScreenDistance": (
+         "REQUIRED",
+         "Screen distance in cm (for example 60 for a screen distance of 60 cm), "
+         'if no screen use "n/a". '
+         "For MRI, it corresponds to the distance between the head-coil mirror "
+         "to the projection screen for example."),
+   }
+) }}
+
+Specific eye-tracking fields SHOULD be present:
+
+{{ MACROS___make_metadata_table(
+   {
+      "IncludedEyeMovementEvents": "RECOMMENDED",
+      "DetectionAlgorithm": "RECOMMENDED",
+      "DetectionAlgorithmSettings": "RECOMMENDED",
+      "StartMessage": "RECOMMENDED",
+      "EndMessage": "RECOMMENDED",
+      "KeyPressMessage": "RECOMMENDED",
+      "CalibrationType": "RECOMMENDED",
+      "CalibrationUnit": "RECOMMENDED",
+      "CalibrationPosition": "RECOMMENDED",
+      "MaximalCalibrationError": "RECOMMENDED",
+      "AverageCalibrationError": "RECOMMENDED",
+      "CalibrationList": "RECOMMENDED",
+      "RecordedEye": "RECOMMENDED",
+      "EyeCameraSettings": "RECOMMENDED",
+      "FeatureDetectionSettings": "RECOMMENDED",
+      "GazeMappingSettings": "RECOMMENDED",
+      "RawDataFilters": "RECOMMENDED",
+      "ScreenRefreshRate": "RECOMMENDED",
+      "AOIDefinition": "RECOMMENDED",
+      "PupilFitMethod": "RECOMMENDED",
+   }
+) }}
+
+### Example
+
+```JSON
+{"TaskName": "PSC_train",
+"InstitutionName": "Goethe-University of Frankfurt; Department of Psychology",
+"InstitutionAdress": "Theodor-W.-Adorno-Platz 6 60323 Frankfurt am Main; Germany",
+"Manufacturer": "SR-Research",
+"ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
+"SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
+"TaskDescription": "Sentence reading",
+"Instructions": "Read sentences as you would read a book or a newspaper",
+"SamplingFrequency": 1000,
+"SampleCoordinateUnit": "pixel",
+"SampleCoordinateSystem": "gaze-on-screen",
+"EnvironmentCoordinates": "top-left",
+"IncludedEyeMovementEvents": [["Start of fixation", "SFIX"],["End of fixation", "EFIX"],["Start of saccade", "SSACC"], ["End of saccade", "ESACC"],["Start of blink", "SBLINK"], ["End of blink", "EBLINK"]],
+"DetectionAlgorithm": "SR-Research",
+"StartMessage": "SENTENCESTART",
+"EndMessage": "SENTENCESTOP",
+"KeyPressMessage": "ANSWER",
+"RecordedEye": "Both",
+"ScreenSize": [38.6, 29],
+"ScreenResolution": [1024, 768],
+"ScreenDistance": 60,
+"AOIDefinition": ["square",["x_start", "x_stop", "y_start", "y_stop"]] ,
+}
```

Comment:
not valid json (trailing `,` in the last item) and better be formatted since it is rendered without reformatting: https://bids-specification--1128.org.readthedocs.build/en/1128/modality-specific-files/eye-tracking.html#example.  Here is how `jq .` does see it (after the fix of `,`):

```suggestion
{
  "TaskName": "PSC_train",
  "InstitutionName": "Goethe-University of Frankfurt; Department of Psychology",
  "InstitutionAdress": "Theodor-W.-Adorno-Platz 6 60323 Frankfurt am Main; Germany",
  "Manufacturer": "SR-Research",
  "ManufacturersModelName": "EYELINK II CL v4.56 Aug 18 2010",
  "SoftwareVersion": "SREB1.10.1630 WIN32 LID:F2AE011 Mod:2017.04.21 15:19 CEST",
  "TaskDescription": "Sentence reading",
  "Instructions": "Read sentences as you would read a book or a newspaper",
  "SamplingFrequency": 1000,
  "SampleCoordinateUnit": "pixel",
  "SampleCoordinateSystem": "gaze-on-screen",
  "EnvironmentCoordinates": "top-left",
  "IncludedEyeMovementEvents": [
    [
      "Start of fixation",
      "SFIX"
    ],
    [
      "End of fixation",
      "EFIX"
    ],
    [
      "Start of saccade",
      "SSACC"
    ],
    [
      "End of saccade",
      "ESACC"
    ],
    [
      "Start of blink",
      "SBLINK"
    ],
    [
      "End of blink",
      "EBLINK"
    ]
  ],
  "DetectionAlgorithm": "SR-Research",
  "StartMessage": "SENTENCESTART",
  "EndMessage": "SENTENCESTOP",
  "KeyPressMessage": "ANSWER",
  "RecordedEye": "Both",
  "ScreenSize": [
    38.6,
    29
  ],
  "ScreenResolution": [
    1024,
    768
  ],
  "ScreenDistance": 60,
  "AOIDefinition": [
    "square",
    [
      "x_start",
      "x_stop",
      "y_start",
      "y_stop"
    ]
  ]
}
```

Reply:
OK for the trailing `,` but I don't understand your comment about "better be formatted".

Reply:
i.e. indented, "pretty printed", etc.  Check other examples of .json files throughout the spec.  Unfortunately I do not think we yet have some consistent formatting enforced for them programmatically for uniform appearance etc.

Reply:
I tried something... 
https://github.com/bids-standard/bids-specification/pull/1128/commits/8fc5f0ffff292d9577c166945080640c3558eaf1